config/redirects.yml

```diff
@@ -427,4 +427,7 @@ redirects:
   "/life-as-a-teacher/become-an-early-years-teacher": "/life-as-a-teacher/age-groups-and-specialisms/early-years-teachers"
   "/blog/becoming-a-mum-sparked-my-interest-in-teaching": "/life-as-a-teacher/teaching-as-a-career/becoming-a-parent-sparked-my-interest-in-teaching"
   "/blog/financiers-future-in-maths": "/life-as-a-teacher/teaching-as-a-career/making-a-career-change-to-teaching"
+  "/blog/abigails-career-progression-story": "/life-as-a-teacher/teaching-as-a-career/abigails-career-progression-story"
```

Comment:
There is a # Blog section in the redirects file - might be better to put these in there (plus the becoming a mum and financiers redirects in 428/9) to keep them all together?